samples/high_throughput_with_dask/run_pv.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import dask.distributed
+import logging
+import numpy as np
+
+import dask_espresso
+
+logging.basicConfig(level=logging.WARN)
+
+
+PYPRESSO = "/data/weeber/es/build/pypresso"
+SIM_SCRIPT = "lj_pressure.py"
+
+N_STEPS = int(2E4)
+N_PARTICLES = 100
+VOLUME_FRACTIONS = np.arange(0.1, 0.52, 0.01)
+
+
+client = dask.distributed.Client(sys.argv[1])
```

Comment:
is the argument theoretically supposed to be either a `Cluster` instance or `None` or is it something different altogether?

Reply:
I made it clear that this is a scheuler address that LocalCluster does not work and clusters with remote workers probably will.